PhysicsTools/NanoAOD/python/run3scouting_cff.py

```diff
@@ -141,7 +141,7 @@
          nValidStripHits = Var('tk_nValidStripHits', 'int', doc='number of valid strip hits'),
          nTrackerLayersWithMeasurement = Var('tk_nTrackerLayersWithMeasurement', 'int', doc='number of tracker layers with measurements'),
          qoverp = Var('tk_qoverp', 'float', precision=10, doc='qoverp'),
-         _lambda = Var('tk_lambda', 'float', precision=10, doc='lambda'),
+         lambda_ = Var('tk_lambda', 'float', precision=10, doc='lambda'),
```

Comment:
why keeping the trailing underscore at all ?

Reply:
```lambda``` is a keyword in python and I believe that was the reason initially to call it ```_lambda```.